manifests/y/Youqu/ToDesk/4.7.6.1/Youqu.ToDesk.installer.yaml

```
@@ -0,0 +1,17 @@
+# Created with YamlCreate.ps1 Dumplings Mod
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Youqu.ToDesk
+PackageVersion: 4.7.6.1
+InstallerType: nullsoft
+Scope: machine
+UpgradeBehavior: install
+Protocols:
+- todesk
+ProductCode: ToDesk
+Installers:
+- Architecture: x86
+  InstallerUrl: https://dl.todesk.com/irrigation/ToDesk_4.7.6.1.exe
+  InstallerSha256: C56DE15801D97D9E4EF297F458CB329AC39C24B7BFB92F489AB59523FE2DDD84

```

```suggestion
  InstallerSha256: B122939731B13F03FD7D0F366B000726D49A0921689C4C881E01D129CC429352
```

(Automated response - build 943.)